locale/it-IT/multipletab/multipletab.dtd

```diff
@@ -109,7 +109,7 @@
 <!ENTITY config.contextMenu.removeLeftTabs  "Chiudi schede a sinistra (sopra)">
 <!ENTITY config.contextMenu.removeRightTabs "Chiudi schede a destra (sotto)">
 
-<!ENTITY config.menuEditor.description "È possibile personalizzare questo menu utilizzando Menu Editor.">
+<!ENTITY config.menuEditor.description "È possible personalizzare questo menu utilizzando Menu Editor.">

```

Comment:
Is this really OK? I cannot determine it because I cannot read any Italian sentence...